yt_dlp/extractor/bilibili.py

```diff
@@ -862,10 +869,16 @@ def _real_extract(self, url):
             self.raise_login_required('This video is for premium members only')
 
         headers['Referer'] = url
-        play_info = self._download_json(
-            'https://api.bilibili.com/pgc/player/web/v2/playurl', episode_id,
-            'Extracting episode', query={'fnval': '4048', 'ep_id': episode_id},
-            headers=headers)
+
+        play_info = self._search_json(
+            r'playurlSSRData\s*?=\s*?', webpage, 'embedded page info', episode_id,
+            end_pattern='\n', default=None)
+        if not play_info:
+            play_info = self._download_json(
+                'https://api.bilibili.com/pgc/player/web/v2/playurl', episode_id,
+                'Extracting episode', query={'fnval': 12240, 'ep_id': episode_id},
+                headers=headers)
```

Comment:
is this change unrelated?

Reply:
they're also related to extracting(without logging in) formats that were previously available only when logged in. could leave them to another pr if you want

Reply:
ah ok we can keep it in this one then